Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM node:alpine
 WORKDIR /app
 
 # Install dependencies
-COPY package.*json ./
+COPY package*.json ./
```

Comment:
This is a bug I found. With this change, package-lock.json is taken into account

Reply:
Good catch. I found a similar transposition in the `.dockerignore` by the same dev. There might be others.